pulp_python/app/serializers.py

```diff
@@ -234,6 +234,18 @@ class Meta:
         model = python_models.PythonPackageContent
 
 
+class MinimalPythonPackageContentSerializer(PythonPackageContentSerializer):
+    """
+    A Serializer for PythonPackageContent.
+    """
+
+    class Meta:
+        fields = core_serializers.ContentSerializer.Meta.fields + (
+            'filename', 'packagetype', 'name', 'version'
+        )
```

Comment:
It looks like PythonPackageContentSerializer uses 'artifact' instead of 'artifacts'. In order to be consistent, what about doing:

```python
fields = tuple(set(core_serializers.ContentSerializer.Meta.fields) - {'artifacts'}) + (
    'filename', 'packagetype', 'name', 'version'
)
```